app/views/directives/osc-image-summary.html

```diff
@@ -4,10 +4,14 @@
   <div ng-show="resource | annotation:'provider'">Provider: {{ resource | annotation:'provider' }}</div>
   <div ng-show="resource.metadata.namespace">Namespace: {{ resource.metadata.namespace }}</div>
 </div>
-<div class="resource-description"
+<div class="resource-description pad-bottom-none"
      ng-bind-html="resource | description | linky"
```

Comment:
So, for "Build and run PHP 5.6 applications", I wouldn't mind if that link, instead of going to the sample repo, was just a link to either [our docs](https://docs.openshift.com/online/using_images/s2i_images/php.html) or to the [SCLorg repo for PHP 5.6](https://github.com/sclorg/s2i-php-container/tree/master/5.6), possibly with a little book icon next to it indicating docs (see Heroku screenshot example below). Linking to the SCLorg would be easier since we could use the same link for all OpenShift flavors, but this might be some kind of annotation on the image stream anyways. Didn't we have a docs link annotation in v2 cartridges?

![Heroku UI, docs link](https://dl.dropboxusercontent.com/u/5246289/Screen%20Shot%202016-08-03%20at%203.11.36%20PM.png)